resources/views/helper.php

```diff
@@ -14,8 +14,11 @@
 <?php if ($namespace == '\Illuminate\Database\Eloquent'): continue; endif; ?>
 namespace <?= $namespace == '__root' ? '' : trim($namespace, '\\') ?> { 
 <?php foreach($aliases as $alias): ?>
-
-    <?= $alias->getClassType() ?> <?= $alias->getExtendsClass() ?> {
+    
+    /**
+     * @mixin <?= $namespace ?>\<?= $alias->getExtendsCLass() ?> 
+     */
```

Comment:
This can't be done like this anymore, because in the meantime the phpdoc of the original class is injected already => https://github.com/barryvdh/laravel-ide-helper/pull/618

@zedentox did you check never versions if they solve your problem?